kolibri/plugins/coach/assets/src/views/groups-page/index.vue

```diff
@@ -79,16 +79,16 @@
     },
     computed: {
       showCreateGroupModal() {
-        return this.modalShown === GroupModals.CREATE_GROUP;
+        return this.groupModalShown === GroupModals.CREATE_GROUP;
```

Comment:
> Again, not sure I understand. What object are you referring to? Here, groupModalShown is set to a constant string, not an object.

I misunderstood the structure before, my mistake.

> I don't understand what you mean - we do have an action to change the state called displayModal in actions/groups.js.

just looked through out modal code, and there isn't a way to do what I had in mind above. My bad again, please disregard.